satpy/tests/reader_tests/test_fci_base.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+
+"""FCI base reader tests package."""
+
+# import datetime as dt
+import unittest
+
+from satpy.readers.fci_base import calculate_area_extent
+from satpy.tests.utils import make_dataid
+
+
+class TestCalculateAreaExtent(unittest.TestCase):
```

Comment:
We try to go over to pytest nowadays, rather that using unittest...
```suggestion
class TestCalculateAreaExtent:
```

Reply:
I have now converted this test to use pytest instead.